public/app/percona/shared/core/reducers/nodes/nodes.utils.ts

```diff
@@ -54,3 +54,25 @@ export const nodeFromDbMapper = (nodeFromDb: NodeDB[]) => {
     } as Node;
   });
 };
+
+export const nodesOptionsMapper = (nodeFromDb: NodeDB[]) => {
```

Comment:
You can specify the return type here `(nodeFromDb: NodeDB[]): NodeOption[]`. You should avoid `as` keyword if possible as it can lead to [problems](https://dev.to/alexanderop/the-problem-with-as-in-typescript-why-its-a-shortcut-we-should-avoid-2km4#the-problem-with-raw-as-endraw-) down the line.

Reply:
fixed